specification/advisor/resource-manager/Microsoft.Advisor/stable/2017-04-19/advisor.json

```diff
@@ -121,29 +117,25 @@
         }
       },
       "put": {
-        "tags": [
-          "Configurations"
-        ],
+        "tags": ["Configurations"],
         "summary": "Create/Overwrite Azure Advisor configuration.",
         "description": "Create/Overwrite Azure Advisor configuration and also delete all configurations of contained resource groups.",
         "operationId": "Configurations_CreateInSubscription",
-        "parameters": [
-          {
-            "$ref": "#/parameters/apiVersionParameter"
-          },
-          {
-            "name": "configContract",
-            "in": "body",
-            "description": "The Azure Advisor configuration data structure.",
-            "required": true,
-            "schema": {
-              "$ref": "#/definitions/ConfigData"
-            }
-          },
-          {
-            "$ref": "#/parameters/subscriptionIdParameter"
+        "parameters": [{
+          "$ref": "#/parameters/apiVersionParameter"
+        },
+        {
+          "name": "configContract",
+          "in": "body",
+          "description": "The Azure Advisor configuration data structure.",
+          "required": true,
+          "schema": {
+            "$ref": "#/definitions/ConfigData"
           }
-        ],
+        },
+        {
+          "$ref": "#/parameters/subscriptionIdParameter"
+        }],
         "responses": {
```

Comment:
response body is missing. when doing PUT on something, the response must contain what was created. Also, the request model and response model must be the same.